server/plugins/connect-plugin-api/src/main/java/com/oceanbase/odc/plugin/connect/api/InformationExtensionPoint.java

```diff
@@ -28,4 +28,6 @@ public interface InformationExtensionPoint extends ExtensionPoint {
 
     String getDBVersion(Connection connection);
 
+    String getODPVersion(Connection connection);
```

Comment:
ODP is a concept specific to OB and is not suitable to be included in a general interface.

Reply:
![image](https://github.com/user-attachments/assets/2a8a2169-2ae0-4d25-b0c2-dafe0a45edac)
As above, put method  `getODPVersion` in OBUtils